base_user_gravatar/tests/test_res_users.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please use short headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py